tools/ovc/openvino/tools/ovc/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -19,6 +19,17 @@ def get_pytorch_decoder(model, example_inputs, args):
     except Exception as e:
         log.error("PyTorch frontend loading failed")
         raise e
+    try:
+        import nncf
+        from nncf.torch.nncf_network import NNCFNetwork
+        from packaging import version
```

Comment:
do we need to add `packaging` dependency?

Reply:
No, that is because user has NNCFNetwork if nncf is installed on his machine.

Reply:
but user installed older NNCF, she / he will not receive error message